plugins/tmdb.py

```diff
@@ -0,0 +1,108 @@
+from collections import OrderedDict
+from urllib.parse import quote_plus
+from . import plugin
+
+import requests
+
+
+class TMDBPlugin(plugin.Plugin):
+    def __init__(self):
+        super().__init__()
+        self._TMDB_API_URL = self._config.get("TMDB_API_URL")
+        self._TMDB_API_KEY = self._config.get("TMDB_API_KEY")
+        self._TMDB_MAX_RESULTS = self._config.get("TMDB_MAX_RESULTS", 3)
+
+    def receive(self, request):
+        if super().receive(request) is False:
+            return False
+
+        if request['text'].lower().startswith("moonbeam movies"):
+            self._log.debug(f"Got movie request: {request['text']}")
+            command = request['text'].split()
+
+            movie = ' '.join(command.split()[2:])
+
+            if not movie:
+                return {
+                    'channel': request['channel'],
+                    'text': 'A movie title must be specified.'
+
+                }
+
+            # Todo check for status -CC
+            api_config = requests.get(
+                f'{self._TMDB_API_URL}/configuration',
+                params={
+                    'api_key': self._TMDB_API_KEY
+                }
+            ).json()
+
+            lookup = requests.get(
+                f'{self._TMDB_API_URL}/search/movie',
+                params={
+                    'api_key': self._TMDB_API_KEY,
+                    'query': quote_plus(movie)
+                }
+            ).json()
+
+            movie_base = 'https://www.themoviedb.org/movie'
+            media_base = api_config['images']['secure_base_url']
+
+            # Get the smallest image, slack will still crop it
+            # maybe add in flight resizing at some point -CC
+            image_size = api_config['images']['poster_sizes'][0]
+
+            result_num = 0
+
+            if lookup['total_result'] == 0:
```

Comment:
Looks like this should be `total_results` (plural), I assume this is a typo.

Reply:
Also correct